src/proxy-ui-api/src/main/resources/openapi-definition.yaml

```diff
@@ -5376,7 +5376,6 @@ components:
       description: service client. May be a subsystem, local group, or a global group
       required:
         - id
-        - subject_type
```

Comment:
The refactored endpoints do not use subject_type for anything, since subject_type can be derived from the encoded id. I wanted to make life easier for the API user and not require subject_type when it is not necessary.